dotnet/csharp.bzl

```diff
@@ -153,7 +153,8 @@ def _csc_get_output(ctx):
 
 def _csc_collect_inputs(ctx, extra_files=[]):
   depinfo = _make_csc_deps(ctx.attr.deps, extra_files=extra_files)
-  inputs = set(ctx.files.srcs) + depinfo.dlls + depinfo.transitive_dlls
+  inputs = set(ctx.files.srcs) + depinfo.dlls + depinfo.transitive_dlls \
```

Comment:
Nit: Enclose with parens instead of using a line splice? Also the indentation on L157 is off.